packages/@okta/vuepress-site/docs/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -6,48 +6,6 @@ You need two values from the Okta application and the Admin Console that you wor
 * **Client ID** &mdash; Find it in the applications list or on the application's **General** tab.
 * **Okta domain** &mdash; You can find the Okta Domain in the Admin Console's global header in the upper-right corner of the page. Click the section that displays your email and company name.  A drop-down box appears and displays general org information including the full Okta domain (for example, subdomain.okta.com).
 
-In your application code, build a config object. This is used to initialize the Okta services with the values specific to your application:
```

Comment:
Oh, I think I understand now. So, you moved this piece from the general content to lang-specific sections. Right?

Reply:
Exactly, but only those relevant for JS but not for Blazor